package.json

```
@@ -10,6 +10,7 @@
     "deploy:dev": "docker buildx build --push --tag stonith404/pingvin-share:development --platform linux/amd64,linux/arm64 ."
   },
   "devDependencies": {
-    "conventional-changelog-cli": "^3.0.0"
+    "conventional-changelog-cli": "^3.0.0",
+    "prettier": "3.3.3"
```

```suggestion
    "conventional-changelog-cli": "^3.0.0"
```
Is there a reason why you've added this dependency?

---

I have vary limited knowledge of javascript but it asked me to install prettier to run the format script so I added it to the dependencies to resolve the install problem.
(I have no idea if this makes sense or not, sorry if it's not 😅)

---

Okay, no problem. :) Then, I’ll simply undo the change.